.github/workflows/cdkactions_docker-django-base.yaml

```diff
@@ -6,6 +6,84 @@ on:
     paths:
       - docker/django-base/**
 jobs:
+  publish-django-base-3-9-14:
```

Comment:
Don't know if this is possible, but is there a way to inject a variable in here based on the python versions specified in `docker.ts`? Seems like copy pasting per python version might get a little annoying later on.

Reply:
To add a python version, you can change the list of python versions supported in `main.ts`:
```
new DjangoBaseDockerStack(app, {pythonVersions: ['3.9.14', '3.8.5', '3.10.1']});
```

Originally, this was a default value in `docker.ts`, but to make it more explicit, I have moved it to `main.ts`.